packages/patternfly-4/react-core/src/components/Title/levels.js

```diff
@@ -0,0 +1,8 @@
+export const HeadingLevels = {
```

Comment:
Looks like you'll want to add this to the index.js file so it can be imported by consumer?

Reply:
Is this a user facing component? and is There a need for the separate levels.js file?